extensions/bitwarden/src/search.tsx

```diff
@@ -0,0 +1,191 @@
+import {
+  ActionPanel,
+  List,
+  Icon,
+  showToast,
+  ToastStyle,
+  PushAction,
+  Detail,
+  CopyToClipboardAction,
+  getPreferenceValues,
+} from "@raycast/api";
+import { Item, Folder } from "./types";
+import { useEffect, useState } from "react";
+import treeify from "treeify";
+import { filterNullishPropertiesFromObject, codeBlock, titleCase } from "./utils";
+import { useBitwarden as useSessionToken } from "./hooks";
+import { TroubleshootingGuide, UnlockForm } from "./components";
+import { existsSync } from "fs";
+import { BitwardenApi } from "./api";
+import { dirname } from "path";
+
+const { cliPath, clientId, clientSecret, fetchFavicons } = getPreferenceValues();
+process.env.PATH = dirname(cliPath)
+const bitwardenApi = new BitwardenApi(clientId, clientSecret);
+
+export default function ListCommand(): JSX.Element {
```

Comment:
```suggestion
export default function List(): JSX.Element {
```
`Command` is implied here.

Reply:
I replaced it by Search to respect the nomenclature defined in the package.json file